src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```
@@ -59,6 +59,10 @@ Copyright (c) .NET Foundation. All rights reserved.
     <GenerateRuntimeConfigurationFilesInputs Include="$(UserRuntimeConfig)" Condition=" Exists($(UserRuntimeConfig)) " />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFrameworks)' != ''">
+    <ProjectConfigurationDescription Include="TargetFramework:$(TargetFramework)"/>
```

We made this switch, right?

```suggestion
    <ProjectConfigurationDescription Include="TargetFramework=$(TargetFramework)"/>
```